keps/sig-network/4410-k8s-network-interface/README.md

```diff
@@ -187,760 +34,42 @@ and accommodate advanced functionalities and potential areas for expansion.
 
 ### Goals
 
-<!--
-List the specific goals of the KEP. What is it trying to achieve? How will we
-know that this has succeeded?
--->
-
 1. Design a cool looking t-shirt
-2. Design and implement the KNI-API
-3. Provide documentation, examples, troubleshooting and FAQ's for KNI.
-   * we should provide a example network runtime
-4. Provide an API that is flexible for experimentation and opinionated use cases
-   * example extradata map[string] string
-5. Provide integration with on premise or cloud systems to provide network status
-6. Provide an API that provides networks available on the node
-7. Determine the reference implementation
-8. Establish feature parity with current [ADD, DEL]
-9. Decouple Node and Pod network setup
-10. Ensure that the network runtime is consolidated inside of a Pod
+2. Make "networks" and their configuration discoverable via the Kubernetes API
+3. Provide highly focused networking APIs that are composable
+3. Support multi-network attachment for `Pods`
```

Comment:
I am wondering if this could be worded differently such as support the ability to add 1 or many network attachments? 

The reverse should be said as well? If we can add 1 or many, we should be able to detach them as well.

Reply:
I like the worded you have in your branch better, let's go with that